app/src/main/java/ca/chronofit/chrono/util/helpers/UtilClass.kt

```diff
@@ -35,4 +38,15 @@ fun formatTime(time: TimeObject, separator: String): String {
     text += df.format(time.seconds) + "."
     text += time.milliseconds.toString() + time.tenMilliseconds.toString()
     return text
+}
+
+fun allocateForDeviceBars(res: Resources, view: ViewGroup) {
+    val navBar = res.getIdentifier("navigation_bar_height", "dimen", "android")
+    val statusBar = res.getIdentifier("status_bar_height", "dimen", "android")
+    val navBarSize = res.getDimensionPixelSize(navBar)
+    val statusBarSize = res.getDimensionPixelSize(statusBar)
+    println("debug: $navBarSize")
```

Comment:
Do you want this in prod? If you do, log it, dont print.

Reply:
Nah it was just there for dev purposes, got rid of it.